core/api.go

```diff
@@ -99,6 +99,7 @@ func (h *Handler) init() {
 	h.actionDecoders = map[string]func(data []byte) (txbuilder.Action, error){
 		"control_account":                h.Accounts.DecodeControlAction,
 		"control_program":                txbuilder.DecodeControlProgramAction,
+		"control_with_receiver":          txbuilder.DecodeControlWithReceiverAction,
```

Comment:
Good question. I actually started (then paused) a PR that would allow the core to accept action labels with prepositions, e.g. "control_with_account", etc., in addition to the ones they accept now. The problem we have now is that the SDKs expose the form with prepositions to the user (because they are way more legible), and the wire protocol doesn't use them. Is that too bikesheddy?